Project-Management/CoreTeam/workspace-notes.md

```diff
@@ -0,0 +1,43 @@
+# Workspace
+
+## Issues with the current workspace
+
+Focussing mainly on `MatrixWorkspace`.
+
+1. Constructed iteratively, i.e., workspaces start their life in an incomplete and dysfunctional state, vital information is added only later (size, instrument, spectrum to detector mapping).
```

Comment:
I think your point 1. Must be fixed in any new design. This default-construct & later initialise pattern used in Mantid is not good.